mzPlot/Annotations/PlotTextAnnotation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OxyPlot;
+using OxyPlot.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace mzPlot
+{
+    public class PlotTextAnnotation : Annotation
+    {
+        public PlotTextAnnotation()
+        { 
+
+        }
+
+        public string Text { get; set; }
+        public double X { get; set; }
```

Comment:
it's not clear to me if this is a data label or just floating text in the graph with offsets. If offsets, what is the reference point from which the offsets are measured.

Reply:
this one is a floating text in the graph, not coupled to any data. I think it's in reference to the upper left pixel in the chart but I'll check and make a comment.

I added it because we had a really hard time adding the peptide sequence in metadraw, and right now we use a hacky workaround where we generate a separate image and tack it onto the graph. it's not great. I want to add a series of annotation options for both data-coupled and non-data-coupled, but in a separate PR.